code/WorkInProgress/Particles.dm

```diff
@@ -1404,7 +1404,7 @@ var/matrix/MS0101 = matrix(0.1, 0, 0, 0, 0.1, 0)
 			return
 
 		for(var/atom/A in view(smoke_size, get_turf(location)))
-			if(istype(A, /obj/particle) || istype(A, /obj/overlay/tile_effect/))
+			if(istype(A, /obj/particle) || istype(A, /atom/movable/light/robust_light))
```

Comment:
Ditto, but this should probably just use `continue_if_overlay_or_effect` - and add the new lighting types to that. See also: all the other checks like this.